### PR TITLE
Update setValue to correctly handle special characters, and unquarantine tests

### PR DESCRIPTION
**Pull request by imurchie**
Some tests are skipped on Travis because they failed at one time. Reduce the number of these.

### CODE REVIEW

**Comment by imurchie** [comment]
Turns out the setValue code was messed up for certain inputs.

File: lib/commands/element.js [hunk 1]
```diff
@@ -150,6 +150,18 @@ commands.setValue = async function (value, el) {
...
+    // possible values of `value`:
+    //   ['some text']
+    //   ['s', 'o', 'm', 'e', ' ', 't', 'e', 'x', 't']
+    //   'some text'
+    if (typeof value === 'string') {
+      // plain string, so make it into an array of characters
+      value = value.toString().split('');
+    } else if (Array.isArray(value)) {
+      // make sure that all the strings inside are a single character long
+      value = _.flatMap(value, (v) => v.split(''));
```
Comment (jlipps) [inline lib/commands/element.js]: 👍  flatMap

File: test/functional/basic/basic-e2e-specs.js [hunk 2]
```diff
@@ -189,7 +189,13 @@ describe('XCUITestDriver - basics', function () {
 
       let urlBar = await driver.elementByClassName('XCUIElementTypeTextField');
       await urlBar.clear();
-      await urlBar.sendKeys(GUINEA_PIG_PAGE);
+
+      try {
+        await urlBar.sendKeys(GUINEA_PIG_PAGE);
+      } catch (err) {
+        let logs = await driver.log('syslog');
```
Comment (jlipps) [inline test/functional/basic/basic-e2e-specs.js]: are we adding this for debugging future failed tests?

Comment (imurchie) [inline test/functional/basic/basic-e2e-specs.js]: It's in during this PR to see why the tests are failing intermittently.

**Pull request merged**